builder/vmware/common/step_create_disks.go

```diff
@@ -44,9 +44,14 @@ func (s *StepCreateDisks) Run(ctx context.Context, state multistep.StateBag) mul
 		diskSizes = append(diskSizes, fmt.Sprintf("%dM", uint64(s.MainDiskSize)))
 	}
 	// Additional disks
+	incrementer := 1
```

Comment:
Nit: incrementer should be defined within the if statement. 

An alternative approach here could be a classic for loop

```go
    for i, j := 0, 1; i <= len(s.AdditionalDiskSize); i++ {                                                                                                                                                                                                                                            
                                                                                                                                                                                                                                                                                                       
      if i+j == 7 {                                                                                                                                                                                                                                                                                    
        j++                                                                                                                                                                                                                                                                                            
      }                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                                       
      path := filepath.Join(*s.OutputDir, fmt.Sprintf("%s-%d.vmdk", s.DiskName, i+j))                                                                                                                                                                                                                  
      diskFullPaths = append(diskFullPaths, path)                                                                                                                                                                                                                                                      
      size := fmt.Sprintf("%dM", uint64(s.AdditionalDiskSize[i]))                                                                                                                                                                                                                                      
      diskSizes = append(diskSizes, size)                                                                                                                                                                                                                                                              
                                                                                                                                                                                                                                                                                                       
    }
```

Reply:
if I define it in the loop it gets reset each loop.

Reply:
Sorry, I wasn't specific on which if :facepalm: 

```
if len(s.AdditionalDiskSize) > 0 {
incrementor :=1 
...
```

Reply:
yeah I got it